tests/e2e/run-scanner-v4.bats

```diff
@@ -468,6 +513,32 @@ EOT
     verify_scannerV2_deployed "${CUSTOM_SENSOR_NAMESPACE}"
     verify_scannerV4_indexer_deployed "${CUSTOM_SENSOR_NAMESPACE}"
     verify_deployment_scannerV4_env_var_set "${CUSTOM_SENSOR_NAMESPACE}" "sensor"
+
+    # Test disabling of Scanner V4.
+    info "Disabling Scanner V4 for Central"
+    "${ORCH_CMD}" -n "${CUSTOM_CENTRAL_NAMESPACE}" \
+      patch Central stackrox-central-services --type=merge --patch-file=<(cat <<EOT
+spec:
+  scannerV4:
+    scannerComponent: Disabled
+EOT
+    )
+
+    info "Disabling Scanner V4 for SecuredCluster"
+    "${ORCH_CMD}" -n "${CUSTOM_SENSOR_NAMESPACE}" \
+      patch SecuredCluster stackrox-secured-cluster-services --type=merge --patch-file=<(cat <<EOT
+spec:
+  scannerV4:
+    scannerComponent: Disabled
+EOT
+    )
+
+    sleep 2m # Give the operator some time to reconcile and the deployments to terminate.
```

Comment:
What I was thinking is instead of waiting for 2 minutes, try every 30 seconds and timeout after ~5 minutes.

But I guess you're right and it's not necessary. It seems that the operator starts the reconcile as soon as the CR is modified, and it removes the scanner deployments soon after.

What takes a long time is for the pods to terminate, but `verify_no_scannerV4_deployed` looks for deployments, not pods. With that into account, I think that 2 minutes is fine.